docs/sources/shared/datasources/tempo-search-traceql.md

```diff
@@ -0,0 +1,103 @@
+---
+headless: true
+---
+
+<!--
+This file documents the Search query type for the Tempo data source. It is available as a public preview.
+
+This shared file is included in these locations:
+/grafana/docs/sources/datasources/tempo/query-editor/index.md
+
+If you make changes to this file, verify that the meaning and content are not changed in any place where the file is included.
+
+Any links should be fully qualified and not relative: /docs/grafana/
+ -->
+
+# Create TraceQL queries using Search
+
+{{% admonition type="Note" %}}
+This new Search query type is available as an experimental feature. This feature is under active development and is not production ready. To try this feature, enable the `traceqlSearch` feature flag in Grafana (read documentation). Grafana Cloud users should contact Grafana Support to enable this feature.
```

Comment:
I'm not sure about the parenthetical here, it doesn't seem that helpful to a reader.

Reply:
Because the link is missing :)

Reply:
```suggestion
This new Search query type is available as an experimental feature. This feature is under active development and is not production ready. To try this feature, enable the `traceqlSearch` feature flag in Grafana (read [documentation](/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/)). Grafana Cloud users should contact Grafana Support to enable this feature.
```